public/app/diarization-html-snippet.html

```
@@ -0,0 +1,97 @@
+      </div>
+    </div>
+  </div>
+
+  <!-- Voiceprint Enrollment -->
+  <div class="voiceprint-panel" data-requires-tier="PRO">
+    <div class="voiceprint-panel__header">
+      <span class="vip-label">🔑 Voiceprint Enrollment</span>
+      <span id="voiceprint-status" class="voiceprint-status voiceprint-status--idle">Not enrolled</span>
+    </div>
+    <div class="voiceprint-panel__actions">
+      <button id="enroll-voiceprint-btn" class="vip-btn vip-btn--primary">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)